atlp-rwanda/team-emma-25-e-comm-backend
# Ft-addToWishlist-#184359086

#### What does this PR do?
The backend needs to handle the addition of items to the buyer's Wishlist, including storing the Wishlist in the buyer's account and sending a confirmation message to the front end.

#### Description of Task to be completed?

- [x] Validate that the buyer is logged in.
- [x] Add the item to the buyer's Wishlist.
- [x] Retrieve the product details from the database.
- [x] Return a confirmation message to the front end."

#### How should this be manually tested?
Use postman for testing this feature by sending  `POST /products/wishlist/add/{id}`
You should send a request with the `Authorization` header with a `token` you get after logging in.

#### What are the relevant pivotal tracker/Trello stories?
STORY_ID: #184359086
